app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -141,10 +141,10 @@ class BrowserToolbarView(
                     false
                 }
 
-                display.progressGravity = if (isCustomTabSession) {
-                    DisplayToolbar.Gravity.BOTTOM
-                } else {
+                display.progressGravity = if (shouldUseBottomToolbar) {
```

Comment:
Since custom tabs also adjust to the user setting now, we should just be checking the toolbar position